Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -1,6 +1,7 @@
 import CLibMongoC
 import Foundation
 @testable import MongoSwift
+@testable import MongoSwiftSync
```

Comment:
do we need this import? I think everything you are extending is in `MongoSwift`

Reply:
nope, removed.